skyportal/models.py

```diff
@@ -263,7 +290,7 @@ class Photometry(Base):
     mag_nearest_source = sa.Column(sa.Float, nullable=True)
     e_mag_nearest_source = sa.Column(sa.Float, nullable=True)
 
-    ## external values
+    # external values
     score = sa.Column(sa.Float, nullable=True)  # RB
```

Comment:
Wonder if we will need a real-bogus score version column.

Reply:
Probably should have a separate table, realbogus, which stores `(detection_id, rb_score, rb_version)`.

The `detection_id` then maps to a `source_id`.